tests/foreman/ui/test_classparameters.py

```diff
@@ -155,90 +265,97 @@ def test_positive_override_checkbox(self):
             3.  Submit the changes.
 
         :expectedresults: Parameter Value overridden with new value.
-
-        :caseautomation: notautomated
-
-        :CaseLevel: System
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_negative_override_checkbox(self):
```

Comment:
Because it contains both invalid steps and has incorrect idea in general. You cannot perform actions (like change default value) for disabled checkbox. That fact verified in `test_negative_edit_parameter_dialog`